Formula/pdfsandwich.rb

```diff
@@ -8,6 +8,7 @@ class Pdfsandwich < Formula
   depends_on "ocaml" => :build
   depends_on "gawk" => :build
```

Comment:
The awk that ships with macOS isn't sufficient?

Reply:
One of the scripts says "# gawk is needed because use of non standard regexp".